bigquery/tests/unit/test_table.py

```diff
@@ -751,3 +751,25 @@ def test__row_from_mapping_w_schema(self):
         self.assertEqual(
             self._call_fut(MAPPING, table.schema),
             ('Phred Phlyntstone', 32, ['red', 'green'], None))
+
+
+class TestRow(unittest.TestCase):
+
+    def test_row(self):
+        from google.cloud.bigquery.table import Row
+
+        VALUES = (1, 2, 3)
+        r = Row(VALUES, {'a': 0, 'b': 1, 'c': 2})
```

Comment:
s/`r`/`row`. From [guide][1]

> Names to Avoid
> - single character names except for counters or iterators

[1]: https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming